src/site/resources/css/site.css

```diff
@@ -134,6 +134,10 @@ a[title="toTop"] {
   z-index: 10;
 }
 
+.poweredBy {
+  display: block;
```

Comment:
> But first, why this?

Without `display: block;`, the `poweredBy` images stack next to each other and the menu becomes too wide. The idea of this style is to make them stack below each other. Ideally, they would be stacked next to and above each other like before. But to make that work, they need to be placed inside some container(`div`) and I couldn't find a way to place them inside a `div`. `site.xml` is pretty strict about that.
```html
...
    </ul>
      <a href="https://github.com/checkstyle/checkstyle" title="GitHub" class="poweredBy">
        <img class="poweredBy"  alt="GitHub" src="images/github_logo_social_coding_outlined.png"     />
      </a>
      <a href="https://twitter.com/checkstyle_java/" title="Twitter" class="poweredBy">
        <img class="poweredBy"  alt="Twitter" src="images/twitter_button.png"     />
      </a>
      <a href="https://stackoverflow.com/questions/tagged/checkstyle" title="Stackoverflow" class="poweredBy">
        <img class="poweredBy"  alt="Stackoverflow" src="images/stackoverflow.jpeg"     />
      </a>
      <a href="https://groups.google.com/forum/#!forum/checkstyle" title="GoogleGroups" class="poweredBy">
        <img class="poweredBy"  alt="GoogleGroups" src="images/groups.png"     />
      </a>
      <a href="https://www.ej-technologies.com/products/jprofiler/overview.html" title="JProfiler" class="poweredBy">
        <img class="poweredBy"  alt="JProfiler" src="https://www.ej-technologies.com/images/product_banners/jprofiler_medium.png"     />
      </a>
      </div>
...
```

![Screenshot from 2023-06-26 12-06-23](https://github.com/checkstyle/checkstyle/assets/23459549/7dbce8a5-348b-4334-98cf-eb5b6ec846dd)

Reply:
Ok.